.gitignore

```diff
@@ -9,3 +9,6 @@
 coverage.txt
 profile.out
 /.glide
+
+# files generated during tests
+config/temp
```

Comment:
Not sure if anything was changed on latest master that would help this as master hasn't changed since this comment? Anyway, it still happens off latest - my local save runs the tests and if I happen to go to git diff while that is running it shows many files in `config/temp`. Not a big problem, but it is still there.